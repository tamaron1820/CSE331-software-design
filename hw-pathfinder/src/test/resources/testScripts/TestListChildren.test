# Test whether list of children are existed correctly

CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

AddEdge graph1 n1 n2 100.0
AddEdge graph1 n1 n3 10.0
AddEdge graph1 n1 n2 5.0

ListChildren graph1 n1