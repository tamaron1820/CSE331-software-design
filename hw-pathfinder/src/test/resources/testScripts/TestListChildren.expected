# Test whether list of children are existed correctly

created graph graph1

added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

added edge 100.000 from n1 to n2 in graph1
added edge 10.000 from n1 to n3 in graph1
added edge 5.000 from n1 to n2 in graph1

the children of n1 in graph1 are: n2(5.0) n2(100.0) n3(10.0)
